client/rest-high-level/src/main/java/org/elasticsearch/client/SecurityClient.java

```diff
@@ -235,14 +234,12 @@ public void getRoleMappingsAsync(final GetRoleMappingsRequest request, final Req
      *
      * @param request the request with the user to enable
      * @param options the request options (e.g. headers), use {@link RequestOptions#DEFAULT} if nothing needs to be customized
-     * @return the response from the enable user call
+     * @return {@code true} if the request succeeded (the user is enabled)
      * @throws IOException in case there is a problem sending the request or parsing back the response
-     * @deprecated use {@link #enableUser(RequestOptions, EnableUserRequest)} instead
      */
-    @Deprecated
-    public EmptyResponse enableUser(EnableUserRequest request, RequestOptions options) throws IOException {
-        return restHighLevelClient.performRequestAndParseEntity(request, SecurityRequestConverters::enableUser, options,
-            EmptyResponse::fromXContent, emptySet());
+    public boolean enableUser(EnableUserRequest request, RequestOptions options) throws IOException {
```

Comment:
Why are we flipping the order of these parameters ?  It can result in a user fixing the deprecated warning only to get a new deprecated warning. Not a big deal, but kinda confusing.

Reply:
This is because we changed the return type in 6.x. The only way to keep the method name was to reorder the params, and this deprecation goes from the backward-param-correct-return-value to the correct-param-correct-return-value and finishes the removal of the incorrect return value, `EmptyResponse`

Reply:
Why do we care about the order of the params ? Is it that bad to leave them "backwards" and avoid another round deprecations ?

Reply:
I guess its not bad to leave them, but its not a ton of work to fix them back to what they were. All of the other public API methods prescribe to this. Changing a return type is pretty rare, so this change is not something we will have people doing often. This fixes it "for good", until we have a new and different API.